example/python/prepare.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+cd $(dirname $0)
+
+# generate iroha lib
+cmake -DSWIG_PYTHON=ON ../..
```

Comment:
Quite error-prone way imo, maybe we can just ask the cmake-build dir from the user?

Reply:
As script parameter?